packages/server-web/src/lib/details-extractor.spec.js

```diff
@@ -0,0 +1,21 @@
+const { extractDetails } = require("./details-extractor");
+
+describe("DetailsExtractor", () => {
+  it(
+    "extracts labels and tracked times",
+    testExtraction(
```

Comment:
You could use `expect(extractDetails("your-test-string")).toMatchSnapshot()` directly.